godot-ffi/src/toolbox.rs

```diff
@@ -35,6 +35,24 @@ macro_rules! static_assert_eq_size {
     };
 }
 
+/// Verifies at compile time that two types `T` and `U` have the same size and alignment.
+#[macro_export]
+macro_rules! static_assert_eq_size_align {
```

Comment:
With this, do we still need `static_assert_eq_size`? Or would other checks also benefit from alignment comparison?